### PR TITLE
Added -verbose=true cli flag

### LINKED ISSUE #60: Let terratag users control log-level
Right now, running `terratag` displays all log, even DEBUG logs.
We should let the user pick the log-level when running `terratag`

### PR DESCRIPTION
**Pull request by shlomimatichin**
Added -verbose=true cli flag, [DEBUG] messages off by default
Note that tfschema RPC clients still bypass this filter, since each creates a logger itself... submitting a patch to tfschema to request this changed.
resolves #60

### CODE REVIEW

**Comment by shlomimatichin** [comment]
this regarding issue #60

File: main.go [hunk 1]
```diff
@@ -11,27 +15,41 @@ import (
...
+	}
+	filter := &logutils.LevelFilter{
+		Levels:   []logutils.LogLevel{"DEBUG", "TRACE", "INFO", "WARN", "ERROR"},
+		MinLevel: logutils.LogLevel("INFO"),
+		Writer:   os.Stderr,
+	}
+	if args.Verbose {
+		filter.MinLevel = logutils.LogLevel("DEBUG")
+	}
+	log.SetOutput(filter)
```
Comment (roni-frantchi) [inline main.go:37]: Perhaps we should extract these to a named function -  
1. It'd give a header to what's being done
2. Perhaps that function would be where we set the log level for tfschema

**Review by roni-frantchi — COMMENTED** [review]
@shlomimatichin looks very good to me.  
Do have one minor comment - address at your discretion.  
Other than that, we should probably update the readme with that flag.

File: main.go (hunk 1 above)
Comment (shlomimatichin) [reply main.go:37]: +1. will do post merge, since #62 continues where this pull request ends.

**Pull request closed without merging**